core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -616,6 +616,37 @@ query.timeout.sec=60
 #pg.daemon.pool=true
 #pg.binary.param.count.capacity=2
 
+################ WAL settings ##################
+
+#Enable support of creating and writing to WAL tables
+#cairo.wal.supported=true
+
+#If set to true WAL becomes the default mode for newly created tables. Impacts table created from ILP and SQL if WAL / BYPASS WAL not specified
+#cairo.wal.enabled.default=false
+
+#If set to true WAL becomes the default mode for newly created tables. Impacts table created from ILP and SQL if WAL / BYPASS WAL not specified
+#cairo.wal.enabled.default=false
```

Comment:
repeated block, comment and property definition

Reply:
removed